frontend/src/context/auth-context.tsx

```diff
@@ -1,27 +1,25 @@
 import React from "react";
 
 interface AuthContextType {
-  githubTokenIsSet: boolean;
-  setGitHubTokenIsSet: (value: boolean) => void;
+  tokenIsSet: boolean;
```

Comment:
making these just `token` might be a bit confusing...maybe `gitProviderToken` or similar?

I'm not sure the current FE changes really set us up for having multiple git providers...we might want to revert the FE changes until we've got it wired up with a GitLab input

Reply:
Got it, keeping changes on FE minimal and saving for future PR